device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceIO.java

```diff
@@ -263,43 +261,11 @@ public void close() throws IOException
                 this.receiveTaskScheduler.shutdown();
             }
 
-            try
-            {
-                this.transport.close(IotHubConnectionStatusChangeReason.CLIENT_CLOSE, null);
-            }
-            catch (DeviceClientException e)
-            {
-                this.state = IotHubConnectionStatus.DISCONNECTED;
-                throw new IOException(e);
-            }
-
+            this.transport.close(IotHubConnectionStatusChangeReason.CLIENT_CLOSE, null);
             this.state = IotHubConnectionStatus.DISCONNECTED;
         }
     }
 
-    // Functionally the same as "close()", but without wrapping any thrown TransportException into an IOException
-    public void closeWithoutWrappingException() throws TransportException
```

Comment:
is this removed because the close will now silently consume any thrown exceptions?

Reply:
It's removed because close no longer throws the exception that we used to wrap in an IOException